source/clear-linux/get-started/virtual-machine-install/vmware-player.rst

```diff
@@ -1,116 +1,466 @@
+    * Click :guilabel:`Next`.
+
+      |vmware-player-03|
+
+      Figure 3: VMware Workstation 14 Player - Select guest operating system type
+
+#.  :guilabel:`Name the Virtual Machine`:
+
+    * Give it a name.
+    * Click :guilabel:`Next`.
+
+      |vmware-player-04|
+
+      Figure 4: VMware Workstation 14 Player - Name virtual machine
+
+#.  :guilabel:`Specify Disk Capacity`:
+
+    * Set the desired disk size.
+    * Click :guilabel:`Next`.
+
+      |vmware-player-05|
+
+      Figure 5: VMware Workstation 14 Player - Set disk capacity
+
+#.  :guilabel:`Ready to Create Virtual Machine`:
+
+    * Click :guilabel:`Customize Hardware...`
+    
+      |vmware-player-06|
+
+      Figure 6: VMware Workstation 14 Player - Customize hardware
+
+    * Select :guilabel:`Processors` > :guilabel:`Virtualization engine`, and 
+      check :guilabel:`Virtualize Intel VT-x/EPT or AMD-V/RVI`.
+
+      |vmware-player-07|
+
+      Figure 7: VMware Workstation 14 Player - Set virtualization engine option
+
+    * Click :guilabel:`Close`.
+
+    * Click :guilabel:`Finish`.
+
+Install Clear Linux into the new VM
+***********************************
+
+#.  Select the newly created VM and click :guilabel:`Play virtual machine`.  
+
+    |vmware-player-08|
+
+    Figure 8: VMware Workstation 14 Player - Power on virtual machine
+
+#.  Follow the :ref:`install-on-target` guide to complete the installation of 
+    |CL|.
+
+#.  After the installation is complete, follow the |CL| instruction to reboot the VM.  
+    This will restart the installer again.    
+
+Reconfigure the VM settings to boot the newly installed Clear Linux
+*******************************************************************
+
+#.  Enable the mouse pointer.
+
+    * Press :kbd:`<CTRL>` + :kbd:`<ALT>`.
+
+#.  Disconnect the CD/DVD to stop it from booting the installer ISO again.
+    
+    * Click :guilabel:`Player`.
+    * Go to :guilabel:`Removable Devices` > :guilabel:`CD/DVD (IDE)` > :guilabel:`Settings`.
+
+      |vmware-player-09|
+
+      Figure 9: VMware Workstation 14 Player - Edit CD/DVD settings
+
+    * Under :guilabel:`Device status`, uncheck :guilabel:`Connected` and :guilabel:`Connect at power on`.
+    * Click `OK`.
+
+      |vmware-player-10|
+
+      Figure 10: VMware Workstation 14 Player - Disconnect CD/DVD
+
+#.  Power off the VM.
+
+    * Click :guilabel:`Player`.
+    * Go to :guilabel:`Power` and select :guilabel:`Shut Down Guest`.
+
+      |vmware-player-11|
+
+      Figure 11: VMware Workstation 14 Player - Power off virtual machine
+
+Enable UEFI boot support
+************************
+
+|CL| needs UEFI support in order to boot.  Enable it by appending the following 
+the end of the VM's :file:`.vmx` file:
+
+  .. code-block:: console
+
+    firmware = "efi"
+
+.. note::
+
+  VMware VM files are typically located in:
+
+  * Linux distros: `/home/username/vmware`
+  * Windows: `C:/\Users/\username/\Documents/\Virtual Machines`
```

Comment:
Also, in windows, this is a hidden file and in order to change it you need to unhide it

Reply:
I verified on my Windows system that it is set to "Don't show hidden files, folders, or drives".  When I browsed to the "Virtual Machines" folder via Explorer, I can see the .vmx file.  Its type is "VMware virtual machine configuration".  I added this extra bit of information into the doc.

Reply:
The problem I had was when I edited the file and tried to save it after adding firmware = "efi", it wouldn't let me save it directly.

Reply:
Another issue I had was when I tried to add the vmware extra software, I couldn't figure out what the cdrom /dev path was and couldn't mount it to add the additional software.  As far as I know I may not be able to add the SW at all to clear linux.